Sources/Public/Controls/AnimatedControl.swift

```diff
@@ -100,6 +109,49 @@ open class AnimatedControl: UIControl {
     super.cancelTracking(with: event)
   }
 
+  #elseif canImport(AppKit)
+  open override func mouseDown(with _: NSEvent) {
+    guard let window = window else { return }
+
+    currentState = .highlighted
+    updateForState()
+
+    // AppKit mouse-tracking loop from:
+    // https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/HandlingMouseEvents/HandlingMouseEvents.html#//apple_ref/doc/uid/10000060i-CH6-SW4
+    var keepOn = true
+    while
+      keepOn,
+      let event = window.nextEvent(
+        matching: [.leftMouseUp, .leftMouseDragged],
+        until: .distantFuture,
+        inMode: .eventTracking,
+        dequeue: true)
+    {
+      let mouseLocation = convert(event.locationInWindow, from: nil)
+      let isInside = isMousePoint(mouseLocation, in: bounds)
+
+      if event.type == .leftMouseUp {
+        keepOn = false
+
+        if isInside {
+          handleMouseUp()
+        }
+      }
+
+      let expectedState = (isInside && keepOn) ? LottieNSLottieControlState.highlighted : .normal
+      if currentState != expectedState {
+        currentState = expectedState
+        updateForState()
+      }
+    }
+  }
+
+  @objc
+  func handleMouseUp() {
```

Comment:
Should this be `open`?

Reply:
I think we can keep this `internal` for now, it's an implementation detail.